app/lib/efile/md/md502_calculator.rb

```diff
@@ -377,6 +382,19 @@ def calculate_line_20
         end
       end
 
+      def calculate_line_22
+        # Earned Income Credit (EIC)
+        if filing_status_mfj? || filing_status_mfs? || @direct_file_data.fed_eic_qc_claimed
+          (@direct_file_data.fed_eic * 0.50).to_i
+        elsif !@direct_file_data.fed_eic_qc_claimed && (filing_status_single? || filing_status_hoh? || filing_status_qw?)
```

Comment:
[DUST] `!@direct_file_data.fed_eic_qc_claimed` does not need to be a part of the conditional here, since all that _do_ have `fed_eic_qc_claimed` are in the other conditional & the rest will come into this elsif conditional.